client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -975,7 +975,7 @@ Undocumented.prototype.readTags = function( fn ) {
 Undocumented.prototype.readTagPosts = function( query, fn ) {
 	var params = omit( query, 'tag' );
 	debug( '/read/tags/' + query.tag + '/posts' );
```

Comment:
we need query parameter .tag because it is used to populate the URL parameter in the GET call.